packages/components/stories/DateTimePicker.js

```diff
@@ -5,6 +5,62 @@ import { IconsProvider } from '../src/index';
 
 import InputDateTimePicker, { DateTimePicker } from '../src/DateTimePickers';
 
+class TestPickerWrapper extends React.Component {
+	static propTypes = {
+		...InputDateTimePicker.propTypes,
+	};
+
+	constructor(props) {
+		super(props);
+
+		this.state = {
+			inputEvalValue: '',
+			selectedDateTime: this.props.selectedDateTime,
+		};
+		this.onSubmitEval = this.onSubmitEval.bind(this);
+		this.onChangeEvalInput = this.onChangeEvalInput.bind(this);
+	}
+
+	onSubmitEval() {
+		this.setState(prevState => ({
+			// eslint-disable-next-line no-eval
+			selectedDateTime: eval(prevState.inputEvalValue),
```

Comment:
eval ?

Reply:
It's just for testing purpose, having a way to define any possible value like InvalidDate, etc.
Check with : `new Date("foo")` :)